packages/@glimmer/compiler/lib/passes/1-normalization/keywords/append.ts

```diff
@@ -251,4 +251,62 @@ export const APPEND_KEYWORDS = keywords('Append')
           })
       );
     },
+  })
+  .kw('helper', {
+    assert(
+      node: ASTv2.AppendContent,
+      state: NormalizationState
+    ): Result<{
+      args: ASTv2.Args;
+    }> {
+      let { args } = node;
+
+      let definition = args.nth(0);
+
+      if (definition === null) {
+        return Err(
+          generateSyntaxError(
+            `{{helper}} requires a helper definition as its first positional parameter, did not receive any parameters.`,
+            args.loc
+          )
+        );
+      }
+
+      if (state.isStrict && definition.type === 'Literal') {
```

Comment:
As discussed, should be

```suggestion
      if (definition.type === 'Literal') {
```